setup.cfg

```diff
@@ -115,7 +117,7 @@ envdir={toxworkdir}/lint
 commands =
     black: black --target-version py36 {posargs:--check --diff} setup.py django_redis/ tests/
     ruff: ruff {posargs:check --show-fixes} django_redis/ tests/
-    mypy: mypy {posargs:--cobertura-xml-report .} django_redis tests
+    mypy: mypy {posargs:--cobertura-xml-report .} django_redis
```

Comment:
please restore this, as also tests should be checked for typing errors so that we can notice breaking changes

Reply:
Sure @WisdomPill ! This was just experimental. Any suggestions to avoid tests directory from coverage report?